src/modules/drawing/components/DrawingExporter.vue

```diff
@@ -37,7 +38,7 @@ function exportDrawing() {
         type = 'application/gpx+xml;charset=UTF-8'
     } else {
         fileName = generateFilename('.kml')
-        content = generateKmlString(projection.value, features)
+        content = generateKmlString(projection.value, features, activeKmlLayerName.value)
```

Comment:
would be better to access the name of the `KMLLayer` stored at `getters.activeKmlLayer` and not store that separately (as said above)